.github/workflows/health_check.yml

```diff
@@ -0,0 +1,41 @@
+name: Health check
+on:
+  schedule:
+    - cron: '0 8 * * *' # Run once a day at 8am UTC
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: ['10', '12', '14']
+    name: Node ${{ matrix.node }} Test
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Install Node Modules
+        run: npm install
+      - name: Build package
+        run: npm run build
+      - name: Run tests
+        env:
+          SANDBOX_API_KEY: ${{ secrets.SANDBOX_API_KEY }}
+        run: npm test
+
+  failure-notification:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: exit 1
+      - name: Health check failure
+        if: always()
+        uses: kpritam/slack-job-status-action@v1
+        with:
+          job-status: ${{ job.status }}
+          slack-bot-token: ${{ secrets.SLACK_BOT_TOKEN }}
+          channel: github-actions
```

Comment:
Work to do to make this work: 
- add SLACK_BOT_TOKEN to secrets
- [creating a slack bot in our slack workspace](https://github.com/abinoda/slack-action#creating-a-slack-app)